src/main/java/nf_core/nf/test/utils/Methods.java

```diff
@@ -45,17 +45,32 @@ public static Map<String, Map<String, Object>> removeNextflowVersion(CharSequenc
     return yamlData;
   }
 
-  //wrapper functions for getAllFilesFromDir with default options
+  // Removed the Key2 entry from the Key1 entry
+  // within the input Version YAML file
+  public static Map<String, Map<String, Object>> removeFromYaml(CharSequence versionFile, String Key1, String Key2) {
```

Comment:
```suggestion
  public static Map<String, Map<String, Object>> removeFromYamlMap(CharSequence versionFile, String Key1, String Key2) {
```

I think this name fits a bit better since you remove a key inside of map in the YAML file and not from the root of the file

Reply:
good point